tests/unit/test__init__.py

```diff
@@ -0,0 +1,137 @@
+import sys
+import pytest
+import types
+import warnings
+from unittest import mock
+import pyarrow as pa
```

Comment:
Imports should be grouped in the following order:

1. Standard library imports.
2. Related third party imports.
3. Local application/library specific imports.

You should put a blank line between each group of imports.

https://peps.python.org/pep-0008/#imports

```suggestion
import types
import warnings
from unittest import mock

import pyarrow as pa
import pytest
```